libafl_qemu/librasan/asan/Cargo.toml

```diff
@@ -42,6 +43,10 @@ libc = ["dep:libc"]
 linux = ["dep:rustix"]
 ## Enable the `baby_mimalloc` allocator
 mimalloc = ["dep:baby-mimalloc"]
+## Support multi-threaded targets by using real synchronization primitives
+multi-threaded = ["dep:spin"]
+## Support single-threaded targets by using dummy synchronization primitives
+single-threaded = ["dep:nospin"]
```

Comment:
`unsafe-single-threaded`?

Reply:
Maybe actually _just_ make this a feature and have the other not be a feature? Or _just_ make `threadsafe` a feature, and leave the other one default. In rust features should be additive if possible